OpenDreamRuntime/DreamConnection.cs

```diff
@@ -212,11 +212,13 @@ public void OutputDreamValue(DreamValue value) {
         if (value.TryGetValueAsDreamObject<DreamObjectSound>(out var outputObject)) {
             ushort channel = (ushort)outputObject.GetVariable("channel").GetValueAsInteger();
             ushort volume = (ushort)outputObject.GetVariable("volume").GetValueAsInteger();
+            int offset = (int)outputObject.GetVariable("offset").UnsafeGetValueAsFloat();
```

Comment:
The DM reference says this value is in seconds. Does this not take a float for fraction-of-a-second values?